components/org.wso2.carbon.identity.api.server.authenticators/org.wso2.carbon.identity.api.server.authenticators.v1/src/main/resources/authenticators.yaml

```diff
@@ -0,0 +1,221 @@
+openapi: 3.0.0
+info:
+  version: "v1"
+  title: 'WSO2 Identity Server - Authenticators API Definition'
+  description: 'This document specifies a **RESTful API** for **WSO2 Identity Server Authenticators**'
+  contact:
+    name: WSO2
+    url: 'http://wso2.com/products/identity-server/'
+    email: architecture@wso2.org
```

Comment:
Shall we change this to `iam-dev@wso2.org` as the `architecture@wso2.org` is deprecated now?

Reply:
fixed with e0d117b57a31fb8167d880e7dfc6c3f603702ff7